docs/user/ml/index.asciidoc

```diff
@@ -98,3 +98,42 @@ image::user/ml/images/outliers.png[{oldetection-cap} results in {kib}]
 
 For more information about the {dfanalytics} feature, see
 {ml-docs}/ml-dfanalytics.html[{ml-cap} {dfanalytics}].
+
+[[xpack-ml-aiops]]
+== AIOps
+
```

Comment:
Since I see "technical preview" in the screenshot, should that callout appear here too?

Reply:
Yup, I added the flag via [7168020](https://github.com/elastic/kibana/pull/138485/commits/71680202d2cd2a5be1f279d1eff056e01e37a4fe).